### PR TITLE
Fix replicate random by printing statistics

### PR DESCRIPTION
**Pull request by parasj**
This PR gates off multipart logic in the case replicate random is used

### CODE REVIEW

**Comment by parasj** [comment]
@sarahwooders Multipart transfer logic was interfering with replicate random. Please review and see if this would interfere with multipart logic.

File: skyplane/cli/cli_internal.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def replicate_random(
     stats = launch_replication_job(
         topo=topo, job=job, debug=debug, reuse_gateways=reuse_gateways, use_bbr=use_bbr, use_compression=False, use_e2ee=True
     )
+    print(stats)
```
Comment (sarahwooders) [inline skyplane/cli/cli_internal.py:69]: please remove

**Review by sarahwooders — APPROVED** [review]
lgtm!

File: skyplane/cli/cli_internal.py (hunk 1 above)
Comment (sarahwooders) [inline skyplane/cli/cli_internal.py:69]: Do you want this print statement?

Comment (parasj) [reply skyplane/cli/cli_internal.py:69]: Yeah we need it to report stats after the transfer

**Pull request merged**